e2e/infrastructure/Listener.spec.ts

```diff
@@ -266,6 +341,62 @@ describe('Listener', () => {
         });
     });
 
+    describe('ModifyMultisigAccountTransaction - Restore multisig Accounts', () => {
```

Comment:
Not sure why I understand the addon of this transaction type in tests ?

Are all transaction types being tested in the `Listener` test ?

Reply:
Well, to test the `BondedTransactionAdded`, 'BondedRemoved` and `cosignatureAdded` event, we need to announce some bonded transactions, and also we need a multisig account with cosigners to be able to co-sign. Having the multisig account restored back to normal account, so the test can be re-run multiple times without errorred out

Reply:
Transactions tests are `in transactionHttpSpec`